app/dataSources/dbFetch.tsx

```diff
@@ -43,6 +43,20 @@ export async function getContestantData(keyPrefix: string): Promise<IContestantD
     return userList
 }
 
+export async function getLeagueConfigurationKeys(): Promise<string[]> {
+    const redis = new Redis({
+        url: process.env.KV_REST_API_URL,
+        token: process.env.KV_REST_API_TOKEN
+    });
+
+    const leagueConfigurationKeys: string[] | null = await redis.keys("league_configuration:*");
```

Comment:
How come we are using `redis.keys()` here as opposed to `redis.scan()` 🤔

Reply:
So, we can do redis.scan. I wasn't sure which we should use.

Reply:
Where did you see that the "recommendation" is scan? 

What do you mean by
> but that does return all the config data
does it?

Reply:
The recommendation: https://upstash.com/docs/redis/sdks/py/commands/generic/keys
<img width="630" alt="image" src="https://github.com/user-attachments/assets/9cecd71f-792f-4d43-b154-2a37cf9dd936" />

It returns all the League Configuration data as an array of the matching league configurations, based on the matching pattern

Reply:
I don't think this last part is true 🤔 
> It returns all the League Configuration data as an array of the matching league configurations, based on the matching pattern

Reply:
You're right. I was wrong because I forgot about the redis.json.get addition in the function before. I'll switch to scan.